src/Avalonia.Visuals/Rendering/DefaultRenderLoop.cs

```diff
@@ -78,12 +78,12 @@ protected void Start()
         /// </remarks>
         protected virtual IDisposable StartCore(Action tick)
         {
-            if (_threading == null)
+            if (_runtime == null)
             {
-                _threading = AvaloniaLocator.Current.GetService<IPlatformThreadingInterface>();
+                _runtime = AvaloniaLocator.Current.GetService<IRuntimePlatform>();
             }
 
-            return _threading.StartTimer(TimeSpan.FromSeconds(1.0 / FramesPerSecond), tick);
+            return _runtime.StartSystemTimer(TimeSpan.FromSeconds(1.0 / FramesPerSecond), tick);
```

Comment:
I've only just noticed that we have both `StartTimer` and `StartSystemTimer`. Are they both necessary?

Reply:
One is for starting a timer on the main event loop. Another is for thread-independent timer that can fire on any thread.

Reply:
👍 makes sense. We should probably call them `StartTimer` and `StartDispatcherTimer` or something and have them on the same interface, but that is something for another PR. I will open an issue.

Reply:
I'm not sure if we want to do that, since `StartTimer` lives in framework runtime support library, it's not tied to particular windowing backend.